code/datums/datumvars.dm

```diff
@@ -922,7 +922,7 @@ body
 			to_chat(usr, "This can only be done to instances of type /mob/living/carbon/human")
 			return
 
-		var/new_species = input("Please choose a new species.","Species",null) as null|anything in all_species
+		var/new_species = input("Please choose a new species.","Species",null) as null|anything in (all_species - "Krampus")
```

Comment:
Because the bag is spawned in New() on the mob, and because krampus isn't intended for use outside of banning people with flavor . It really shouldn't be used otherwise.

I'll move everything to the species datum when I next have free time again.